src/components/common/LikelyToGiveSelect/LikelyToGiveSelect.test.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { LikelyToGiveEnum } from 'src/graphql/types.generated';
+import { LikelyToGiveSelect } from './LikelyToGiveSelect';
+
+jest.mock('src/utils/functions/getLocalizedLikelyToGive', () => ({
```

Comment:
Usually, we don't mock simple utilities like this. Can you get it to pass without mocking?